helpers/featureSelect.mjs

```diff
@@ -0,0 +1,85 @@
+import fs from 'fs';
```

Comment:
We have a [`fs-next`](https://github.com/jprichardson/node-fs-extra#nodejs-fs-extra) package available, so you can use its [`removeSync`](https://github.com/jprichardson/node-fs-extra/blob/master/docs/remove-sync.md) and [`emptyDirSync`](https://github.com/jprichardson/node-fs-extra/blob/master/docs/emptyDir-sync.md) methods directly